plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/rest/TestIcebergRestCatalogConnectorSmokeTest.java

```diff
@@ -182,6 +184,25 @@ public void testRegisterTableWithMetadataFile()
                 .hasMessageContaining("metadata location for register_table is not supported");
     }
 
+    @Override
```

Comment:
Why not:
```java
    @Override
    public void testRegisterTableWithTrailingSpaceInLocation()
    {
        assertThatThrownBy(super::testRegisterTableWithTrailingSpaceInLocation)
                .hasMessageContaining("registerTable is not supported for Iceberg REST catalog");
    }
```

Then, you don't need to change base `getTableLocation` to protected.